src/sentry/models/organization.py

```diff
@@ -329,6 +329,48 @@ def default_owner_id(self):
             self._default_owner_id = owners[0].id
         return self._default_owner_id
 
+    @classmethod
+    def _get_bulk_owner_ids(cls, organizations: Collection[Organization]) -> dict[int, int]:
+        from sentry.models.organizationmember import OrganizationMember
+
+        owner_id_table: dict[int, int] = {}
+        org_ids_to_query: list[int] = []
+        for org in organizations:
+            default_owner = getattr(org, "_default_owner", None)
+            if default_owner:
+                owner_id_table[org.id] = default_owner.id
+            else:
+                org_ids_to_query.append(org.id)
+
+        if org_ids_to_query:
+            queried_owner_ids = OrganizationMember.objects.filter(
+                organization_id__in=org_ids_to_query, role=roles.get_top_dog().id
+            ).values_list("organization_id", "user_id")
+
+            for (org_id, user_id) in queried_owner_ids:
+                # An org may have multiple owners. Here we mimic the behavior of
+                # `get_default_owner`, which is to use the first one in the query
+                # result's iteration order.
+                if org_id not in owner_id_table:
+                    owner_id_table[org_id] = user_id
+
+        return owner_id_table
+
+    @classmethod
+    def get_bulk_owner_contacts(
+        cls, organizations: Collection[Organization]
+    ) -> dict[int, RpcUserContact]:
+        owner_id_table = cls._get_bulk_owner_ids(organizations)
+
+        contacts = user_service.get_bulk_contact_info(user_ids=list(owner_id_table.values()))
```

Comment:
This safe to roll out at the same time as the new service method, because nothing is going to hit `get_bulk_owner_contacts` until https://github.com/getsentry/getsentry/pull/13623 is merged.